### DISCUSSION TITLE
Add a plot showing exposure time vs flux on "PFS Targets & Proposal Lists"

### DISCUSSION THREAD

**Discussion by yukimoritani** [Subaru-PFS/spt_target_uploader#154] [Ideas]
To judge feasibility of programs, it is helpful to see achieved S/N on the uploaded targets.
For example, a plot of exposure time vs flux of the targets could be an option. If the plot also shows model lines for a few sets of S/N, it will be nice.

**Comment by monodera** [comment]
I agree to make such plots, but calculating S/N needs ETC  and the computational time would not be negligible. I heard that Takagi-san and Yabe-san are discussing to make a simplified version of ETC to estimate S/N at the selected wavelength. 

Before putting everything into the uploader, we should think about which functionalities are essential for users as a "target uploader". 

Personally, running ETC in the uploading process would be computationally too expensive for users. We can post process the input target list for the technical review. Additionally, including PPP in the uploader is nice, but I feel that it's already introducing some confusion even within the obsproc team

**Comment by monodera** [comment]
Sorry, i a bit misread the suggestion. I agree that it would be nice to have some plots related to flux and exptime in addition to the pointing information.
